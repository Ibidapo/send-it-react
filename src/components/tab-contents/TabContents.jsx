import React from 'react';
import PropTypes from 'prop-types';

import './styles.scss';

const TabContents = ({ children, id }) => {
  return (
    <div className='tab-contents-area' id={id}>
      {children}
    </div>
  );
}

TabContents.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
  id: PropTypes.string
};

export default TabContents;