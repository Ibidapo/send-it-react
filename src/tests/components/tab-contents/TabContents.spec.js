import React from 'react';
import { shallow } from 'enzyme';

import TabContents from '../../../components/tab-contents/TabContents';

let wrapper;

describe('Test for Header Component', () => {
  it('Should render correctly', () => {
    const props = { children: 'element' , id: 'dashboard' };
    wrapper = shallow(<TabContents {...props} />);
    expect(wrapper).toMatchSnapshot();
  })
});