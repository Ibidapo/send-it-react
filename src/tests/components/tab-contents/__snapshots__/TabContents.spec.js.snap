// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Header Component Should render correctly 1`] = `
<div
  className="tab-contents-area"
  id="dashboard"
>
  element
</div>
`;
